app/views/documents/_header.html.erb

```diff
@@ -19,6 +19,9 @@
 <div class="heading-extra">
   <div class="inner-heading">
     <%= render partial: 'shared/available_languages', locals: {object: document} %>
+    <% if @document.respond_to?(:logo_url) && @document.logo_url %>
+      <%= image_tag @document.logo_url, id: "logo-image" %>
```

Comment:
I think it would be good accessibility to also include alt text here, especially as the logo image contains text.